.env.template

```diff
@@ -0,0 +1 @@
+CONFIG_FILE=./openshift/3scale_backend.conf
```

Comment:
I found there's a `openshift/env.test` file. I think it's better to use that file as example and remove this one.

`env.test` is being used to provide env vars to the container, check [openshift/README.md](https://github.com/3scale/apisonator/blob/master/openshift/README.md)